files/index.d.ts

```diff
@@ -1013,8 +1013,8 @@ Notes:
 
 */
 // @SINGLE_MARKER
-export function endsWith(target: string, iterable: string): boolean;
-export function endsWith(target: string): (iterable: string) => boolean;
+export function endsWith<T extends string>(target: T, str: string): str is `${string}${T}`;
```

Comment:
Changed second arg from `iterable` to `str` to match `startsWith` method

Reply:
I think we also need to change description of these methods from `iterable` to something else (`input`?)